### ISSUE TITLE
create ability to specify settings file

### ISSUE THREAD

**Issue by oleksandrzelentsov** [oleksandrzelentsov/snap-sink#1]

**Comment by oleksandrzelentsov** [comment]
solved by 77f0fae8ddfdd9f73fb068d1dffff528e9e58cc9

**Comment by oleksandrzelentsov** [comment]
the option doesn't work so far
use case:

```
$ sink --settings file.conf
... <using specified file>
$ sink
... <using ~/.sync.conf>
```

**Comment by oleksandrzelentsov** [comment]
finally solved by c479f19
